buildSrc/src/main/kotlin/myPackage/Modules.kt

```diff
@@ -0,0 +1,4 @@
+object Modules {
+    const val SHARED = ":shared"
```

Comment:
Correct me if I am wrong but this `const val SHARED` is not used anywhere, right? Can we remove it? 🤔

Reply:
Good catch 😄